tests/test_client.py

```diff
@@ -14,15 +15,21 @@ async def hello_world(scope, receive, send):
     )
 
 
-def test_explicit_base_url():
-    client = TestClient(hello_world, base_url="http://testserver:321")
-    response = client.get("/")
```

Comment:
I guess we can just remove this entire module, and the definition of the TestClient as well, right? We don't want to be re-testing HTTPX. :)

Reply:
done